scene/gui/rich_text_label.cpp

```diff
@@ -2324,7 +2326,7 @@ Error RichTextLabel::append_bbcode(const String &p_bbcode) {
 			pos = brk_end + 1;
 			tag_stack.push_front("font");
 
-		} else if (tag.begins_with("fade")) {
+		} else if (bbcode == "fade") {
 			Vector<String> tags = tag.split(" ", false);
```

Comment:
I'm not sure what you mean here...
These lines are already like that right?
`bbcode == "fade"`

Reply:
Line 2330 (and others like it) where I originally had `Vector<String> tags = tag.split(" ", false)`. That can be removed now because we're already splitting the tags before the if block by spaces. That line can be removed now and all instances of `tags` inside that if-block can now be `split_tag_block`. This is to avoid redundant operations wherever possible. :)

Reply:
Oh ok I understand.
This is done!